include/common/aarch32/el3_common_macros.S

```diff
@@ -195,6 +195,18 @@
 	cps	#MODE32_mon
 	isb
 
+#if SP_MIN_WITH_SECURE_FIQ
+	/* -----------------------------------------------------
+	 * FIQs are secure interrupts trapped by Monitor and non
+	 * secure is not allowed to mask the FIQs.
+	 * -----------------------------------------------------
+	 */
+	ldcopr	r0, SCR
+	orr	r0, r0, #SCR_FIQ_BIT
+	bic	r0, r0, #SCR_FW_BIT
+	stcopr	r0, SCR
+#endif
+
```

Comment:
The `el3_entrypoint_common` macro is also used by BL1. This code would be better placed in `sp_min_entrypoint`.